examples/DatePicker.basic.jsx

```diff
@@ -3,16 +3,45 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 import * as React from 'react';
-import { DatePicker } from '@itwin/itwinui-react';
+import { DatePicker, IconButton, Popover } from '@itwin/itwinui-react';
+import { SvgCalendar } from '@itwin/itwinui-icons-react';
 
 export default () => {
   const [currentDate, setCurrentDate] = React.useState(new Date());
+  const [visible, setVisible] = React.useState(false);
   return (
-    <DatePicker
-      date={currentDate}
-      onChange={(date) => {
-        setCurrentDate(date);
-      }}
-    />
+    <Popover
+      content={
+        <DatePicker
```

Comment:
is adding the content inline like this preferred over setting it to a variable? given that its being pass through props, I wonder if it'd be more readable to separate it via variable.  genuine syntactic question

Reply:
I am kind of going back and forth on this. On the one hand, it makes sense to me for all of the JSX to exist in the return statement, so that people don't have to dig through variables to find the JSX that they are looking for. On the other hand, these examples are supposed to be about the `DatePicker` components specifically, so isolating them in a variable could help to distinguish them from the `Popover` boilerplate.